tests/model_fields/tests.py

```diff
@@ -297,6 +298,16 @@ def test_empty_iterator_choices(self):
         self.assertEqual(WhizIterEmpty(c="").c, "")  # Empty value
 
 
+class GetFieldDisplayGeneratedTests(TestCase):
```

Comment:
I believe this test would be best placed in test_generatedfield.py?

Reply:
Now the logic has been moved within the GeneratedField definition, I agree this is a better place for it